docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -95,6 +95,26 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
       vars:
            app_path: "{{ base_path }}/22"
 
+.. _boolean_variables:
+
+Boolean variables
+=================
+
+Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False``, and so on.
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+
+.. table::
+   :class: documentation-table
+
+   ==================================================================================== ====================================================================
+    Valid values                                                                         Description
+   ==================================================================================== ====================================================================
+   ``true`` , ``True`` , ``t`` , ``yes`` , ``y`` , ``on`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+
+   ``false`` , ``False`` , ``f`` , ``no`` , ``n`` , ``off`` , ``'0'`` , ``0`` , ``0.0``   Falsy values
```

Comment:
`True` and `False` in this context are 'actual boolean values' while `'true'` and `'false'` are strings (why there is a duplication).